src/pineko/fonll.py

```diff
@@ -224,9 +222,11 @@ def scheme(self):
 FNS_CONFIG = [
```

Comment:
Now `MIXED_FNS_CONFIG` and `FNS_CONFIG` should coincide no?

Reply:
The only difference between the two is the `delta_pto`. Maybe we can get rid of on of the two